app/tasks/doctor.php

```diff
@@ -90,32 +90,55 @@
         \sleep(0.2);
 
         try {
-            Console::log("\n" . 'Checking connectivity...');
+            Console::log("\n" . '[Connectivity]');
         } catch (\Throwable $th) {
             //throw $th;
         }
 
-        try {
-            $dbPool = $register->get('dbPool'); /* @var $dbPool DatabasePool */
-            $database = $dbPool->getConsoleDB();
-            $pdo = $dbPool->getPDO($database);
-            Console::success('Database............connected 👍');
-        } catch (\Throwable $th) {
-            Console::error('Database.........disconnected 👎');
-        }
+        $pools = $register->get('pools'); /** @var \Utopia\Pools\Group $pools */
 
-        try {
-            $register->get('cache');
-            Console::success('Queue...............connected 👍');
-        } catch (\Throwable $th) {
-            Console::error('Queue............disconnected 👎');
+        $configs = [
+            'Console.DB' => Config::getParam('pools-console'),
```

Comment:
any reason to use the dot in the key names?  We haven't used this convention before

Reply:
I felt a space would be weird here because of the format, and camelCase would indicate a code concept while this should be a human readable format 🤔